clouddriver-security/src/main/groovy/com/netflix/spinnaker/clouddriver/security/resources/ServerGroupsNameable.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 Google, Inc.
```

Comment:
2018

Reply:
See your other comment - we don't usually update this after file creation - it was just moved.